pkg/kubectl/cmd/portforward_test.go

```diff
@@ -85,9 +85,8 @@ func TestPortForward(t *testing.T) {
 			ff.pfErr = fmt.Errorf("pf error")
 		}
 		cmd := &cobra.Command{}
-		podPtr := cmd.Flags().StringP("pod", "p", "", "Pod name")
-		*podPtr = "foo"
```

Comment:
If we support both means to specify pods, we should test both means of specifying pods.